frontend/src/scenes/experiments/ExperimentView/components.tsx

```diff
@@ -580,12 +580,17 @@ export function PageHeaderCustom(): JSX.Element {
 }
 
 export function ShipVariantModal({ experimentId }: { experimentId: Experiment['id'] }): JSX.Element {
-    const { experiment, sortedWinProbabilities, isShipVariantModalOpen } = useValues(experimentLogic({ experimentId }))
+    const { experiment, isShipVariantModalOpen } = useValues(experimentLogic({ experimentId }))
     const { closeShipVariantModal, shipVariant } = useActions(experimentLogic({ experimentId }))
     const { aggregationLabel } = useValues(groupsModel)
 
     const [selectedVariantKey, setSelectedVariantKey] = useState<string | null>()
-    useEffect(() => setSelectedVariantKey(sortedWinProbabilities(0)[0]?.key), [sortedWinProbabilities(0)])
+    useEffect(() => {
+        if (experiment.parameters?.feature_flag_variants?.length > 0) {
```

Comment:
this check can be safer incase there's an error somewhere in our validation logic, so this doesn't break
```suggestion
        if (experiment.parameters?.feature_flag_variants?.length > 1) {
```